src/app/components/Navigation.tsx

```diff
@@ -6,7 +6,8 @@ import {
 } from 'react-bootstrap'
 
 import NavigationLink from './NavigationLink'
-const logo = require('../bc_logo_transparent.png');
+// const sheriffLogo = require('../logo.svg');
+// const bcLogo = require('../bc_logo_transparent.png');
```

Comment:
Could probably get rid of the commented bits.

Reply:
Done